src/Athena.Importer/data/offering/mthm280-p100-f2018.json

```diff
@@ -0,0 +1,38 @@
+{
+  "id": "c22cd6a4-b7f2-437e-bf45-09e774ae97cd",
```

Comment:
An offering with the same ID exists for `@{id=9dc68c84-69fa-4ace-b8da-d08b44f0cc82; name= ECNM 152 - MICROECONOMICS;}`

Reply:
ok